### PR TITLE
SWC-6410 - fix styling issues

### PR DESCRIPTION
**Pull request by nickgros**
- Fix styling within download cart by dropping some bootstrap
- Always use white for contrast text

### CODE REVIEW

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx [hunk 1]
```diff
@@ -67,7 +65,7 @@ export const DownloadCartPage: React.FunctionComponent<
     refetch()
   }
   return (
-    <div className="DownloadCartPage bootstrap-4-backport">
```
Comment (nickgros) [inline packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx:70]: This `bootstrap-4-backport` was messing up the new card styling, so narrow the scope

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx [hunk 2]
```diff
@@ -169,7 +167,11 @@ export const DownloadCartPage: React.FunctionComponent<
                         helpUrl="https://help.synapse.org/docs/Downloading-Data-From-the-Synapse-UI.2004254837.html"
                       />
                     </div>
-                    <Typography variant={'body1'}>
+                    <Typography
+                      variant={'body1'}
+                      component={'div'}
```
Comment (nickgros) [inline packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx:172]: I wanted to just do `component={'ul'}` but apparently we have this style in SWC:
```css
ol, ul {
  margin-bottom: 0px !important;
}
```
so we'll wrap in a div

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx [hunk 3]
```diff
@@ -189,45 +191,50 @@ export const DownloadCartPage: React.FunctionComponent<
                         </li>
                       </ul>
                     </Typography>
-                    <span>
-                      {data.numberOfFilesAvailableForDownloadAndEligibleForPackaging >
-                        0 && (
+                    {data.numberOfFilesAvailableForDownloadAndEligibleForPackaging >
+                      0 && (
+                      <Button
```
Comment (nickgros) [inline packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx:196]: Use MUI and drop wrapping span

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx [hunk 4]
```diff
@@ -85,18 +86,16 @@ export const DownloadListActionsRequired: React.FunctionComponent<
...
-            if (item) {
-              return renderActionRequired(item)
-            } else return false
-          })}
-          {/* To trigger loading the next page */}
-          <div ref={ref} />
-        </div>
-      )}
-      {isLoading && <LoadingActionRequiredCard />}
+      <Box sx={{ pt: 5 }} display="flex" flexDirection="column" gap={3}>
```
Comment (nickgros) [inline packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx:89]: Move the top padding here from scss and add a gap between cards using flex

File: packages/synapse-react-client/src/lib/utils/theme/palette/Palettes.ts [hunk 5]
```diff
@@ -22,6 +22,7 @@ export const generatePalette = (
     dark:
       (specificColors && specificColors[600]) ||
       tinycolor(mainColor).saturate(5).darken(4).toString(), // 600
+    contrastText: '#fff',
```
Comment (nickgros) [inline packages/synapse-react-client/src/lib/utils/theme/palette/Palettes.ts:25]: All of our colors so far should use white, and some recent change caused many of them to use black (even in SWC). So let's just hard-code here and override if/when we need to

Comment (nickgros) [reply packages/synapse-react-client/src/lib/utils/theme/palette/Palettes.ts:25]: |Before|After|
|---|---|
| image | |

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx [hunk 6]
```diff
@@ -85,18 +86,16 @@ export const DownloadListActionsRequired: React.FunctionComponent<
...
+        {allRows.map((item: ActionRequiredCount) => {
+          if (item) {
+            return renderActionRequired(item)
+          } else return false
+        })}
+        {/* To trigger loading the next page */}
+        {allRows.length > 0 && <div ref={ref} />}
+        {isLoading && <LoadingActionRequiredCard />}
```
[the remark this answers is not in the document]
Comment (nickgros) [reply packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx:97]: |Before|After|
|---|---|
|image||

File: packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx
Comment (nickgros) [inline packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx:1]: |Before|After|
|---|---|
|image||

Comment (jay-hodgson) [reply packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx:1]: Looks virtually identical, nice job

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**